pkg/controller/ground.go

```diff
@@ -418,6 +419,15 @@ func (op *GroundControl) createKluster(kluster *v1.Kluster) error {
 		Region:     region,
 	}
 
+	if err := op.Clients.OpenstackAdmin.CreateContainer(
+		fmt.Sprintf(etcd_util.EtcdBackupStorageContainer, kluster.GetName(), kluster.GetUID()),
```

Comment:
On closer look maybe we should move this format call into a function in the etcd util package and use the function in both places:
`func DefaultStorageContainer(kluster *v1.Kluster) string`